.github/workflows/dev.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
       build_container: 'node:14-buster'
       deploy_container: 'ubuntu:20.04'
       install_packages_command: 'yarn install'
-      build_command: 'yarn build'
+      build_command: 'yarn build:www'
```

Comment:
Why not `release:www`?

Reply:
1. They use in their Docker file `build:www` to build static files.
2. In command `build:release:www` they are trying to upload something to IPFS what is not required to us.